scripts/Makefile

```
@@ -69,4 +69,9 @@ unmount-s3: publish-artifacts
 import-GPG-key:
 	@printf %s $(GPG_PRIVATE_KEY_BASE64) | base64 --decode | gpg --batch --import --no-default-keyring --keyring $(GPG_KEY_RING) -
 
-.PHONY: mount-s3 mount-s3-check publish-artifacts prepare-schema unmount-s3 import-GPG-key
\ No newline at end of file
+clear-fastly-cache: unmount-s3
+ifdef FASTLY_KEY
+	@/usr/bin/curl -i -X POST -H Fastly-Key:$(FASTLY_KEY) https://api.fastly.com/service/$(FASTLY_SERVICE_ID)/purge_all
```

I'd add quotes around this just in case the key has spaces
```suggestion
	@/usr/bin/curl -i -X POST -H "Fastly-Key: $(FASTLY_KEY)" https://api.fastly.com/service/$(FASTLY_SERVICE_ID)/purge_all
```